<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TriangulationThresholdDialog</class>
 <widget class="QDialog" name="TriangulationThresholdDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clout point triangulation Filter</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>172</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Triangle Mean Size Threshold</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="triangleTh_edit">
      <property name="toolTip">
       <string>Max triangle size squared (Perimeter/3) relative to mean size squared</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Coordinates Mean Delta</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Delta X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dxTh_edit">
        <property name="toolTip">
         <string>Max difference in X axis relative to mean delta X</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Delta Y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dyTh_edit">
        <property name="toolTip">
         <string>Max difference in Y axis relative to mean delta Y</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Delta Z</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dzTh_edit">
        <property name="toolTip">
         <string>Max difference in Z axis relative to mean delta Z</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>168</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="toolTip">
       <string>Accept selected values</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="toolTip">
       <string>Cancel selected values - all ignored</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
